src/metrics/loc.rs

```diff
@@ -377,20 +377,21 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_blank_python() {
+    fn test_blank() {
         check_metrics!(
-            "\na = 42\n\n",
+            "
+            a = 42
+            \n",
```

Comment:
Why are you removing `\n` sometimes and keeping it other times?
I'd prefer to be consistent and either always have `\n` or never have it.